color_refinement.py

```diff
@@ -213,7 +213,7 @@ def is_isomorphisms(g: Graph, h: Graph) -> bool:
     elif preprocessing.is_tree(h):
         return False
     else:
-        is_potential_isomorph, g, h, factor = modular_decomposition(g, h)
+        is_potential_isomorph, g, h, factor, md_iso_groups_g, md_iso_groups_h = modular_decomposition(g, h)
```

Comment:
@ClaudiaReuvers hier zie je hoe je het kunt gebruiken. Natuurlijk wil je niet vaker dan nodig `modular_decomposition` aanroepen, dus alleen hier.

Reply:
Uhm hoe weet je nu of een module uit `md_iso_groups_g` isomorph is aan een module in `md_iso_groups_h`?

Reply:
Alle modules binnen 1 groep van de `[[Vertex]]` (dus binnen `md_iso_groups_g`) zijn isomorf.

Reply:
Ik weet alleen niet zeker of je de garantie hebt dat de groepen voor `g` en die voor `h` ook in dezelfde volgorde staan. Even checken.

Reply:
Als ik het goed heb hebben @dorienmc en @MinThaMie  tree refinement nu gemaakt dat er een lijst `[module]` in gaat, waarbij `module = [modules]`. toch?